Source/g_game.c

```diff
@@ -660,6 +667,37 @@ static void G_DoLoadLevel(void)
     }
 }
 
+extern int ddt_cheating;
+
+static void G_ReloadLevel(void)
+{
+  int i;
+
+  if (demorecording || netgame)
+  {
+    gamemap = startmap;
+    gameepisode = startepisode;
+  }
+
+  if (demorecording)
+  {
+    ddt_cheating = 0;
```

Comment:
Why do we need to reset `ddt_cheating` here?

Reply:
Because we want to restart the map in a clean non-cheated state.